src/mesh/smoothing.rs

```diff
@@ -74,12 +74,13 @@ pub fn laplacian_smoothing(
         }
     }
 
-    // FIXME: Calculate smooth normals for the result once we support them
```

Comment:
Do we also want to do `loop_subdivision` in this PR? (no is also an answer)

Reply:
Hmmm, yes, let's